config/config_mgmt.py

```diff
@@ -0,0 +1,709 @@
+#!/usr/bin/env python
+#
+# config_mgmt.py
+# Provides a class for configuration validation and for Dynamic Port Breakout.
+
+try:
+
+    # import from sonic-cfggen, re use this
+    from imp import load_source
+    load_source('sonic_cfggen', '/usr/local/bin/sonic-cfggen')
+    from sonic_cfggen import deep_update, FormatConverter, sort_data
+    from swsssdk import ConfigDBConnector, SonicV2Connector, port_util
+    from json import load
+    from os import system
+    from time import sleep as tsleep
+
+    import sonic_yang
+    import syslog
+    import re
```

Comment:
Suggest to reorder these imports as follows, unless this ordering causes issues:

```
    import re
    import syslog
    from imp import load_source
    from json import load
    from os import system
    from time import sleep as tsleep

    import sonic_yang
    from swsssdk import ConfigDBConnector, SonicV2Connector, port_util

    # <Add a clear comment here describing why you are doing this>
    load_source('sonic_cfggen', '/usr/local/bin/sonic-cfggen')
    from sonic_cfggen import deep_update, FormatConverter, sort_data
```


    
    
```

Reply:
Sure, Let me try and test this.

Reply:
Done